### DISCUSSION TITLE
Existing Plugins?

### DISCUSSION THREAD

**Discussion by otagayan-dev** [reearth/reearth-visualizer#111] [Q&A] [answered]
Are there any existing plugins?

**Comment by HideBa** [comment, marked as answer]
@guesschronic6 

Thank you for asking that.

Now we have a sample plugin [here](https://github.com/rot1024/reearth-plugin-test) one of our team members developed.
We are preparing documents for plugin developers.
When it's ready, we'll officially announce it to all.
If you are interested in that, we are glad.

**Reply by HideBa** [reply]
@guesschronic6 

We've made additional plugins examples [https://github.com/reearth/plugin-example/tree/finished](here).

For your information.
